mautic/mautic
# [UI] fix: Form creation on mobile @RCheesley

13826



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #13826 



## Description

This pull request addresses the misalignment issue of panel headers during form creation on mobile devices. Previously, the content appeared buggy, impacting user experience. 

The fix ensures that the headers are correctly aligned, enhancing the overall usability of the form creation feature on mobile. This improvement benefits users who rely on mobile devices for form management, providing a more consistent and professional interface.





| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/8124fefb-81b0-497d-bf87-4dcda3aa24ab)                                      | 
![image](https://github.com/user-attachments/assets/d9d85a0c-e3b2-4731-8127-b3f65423a9b9)

---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Activate mobile view, open the Forms page, create a new form and see the header for each panel
